models/PartnerBillingTierPerLoan.v1.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "PartnerBillingTierPerLoan",
+  "type": "object",
+  "description": "Structure de facturation du partenaire, par type de prêt et par intervalles de montants de prêts. Si aucun type de prêt n'est précisé, la structure s'appliquera à tous les types de prêts.",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Type de prêt.",
+      "enum": [
+        "FREE_LOAN",
+        "BRIDGE_LOAN",
+        "PTZ_LOAN",
+        "BOSS_LOAN"
+      ]
+    },
+    "tiers": {
+      "type": "array",
+      "items": {
+        "$ref": "./PartnerBillingModel.v1.json"
```

Comment:
Shouldn't this datamodel be renamed something like PartnerBillingTier then? "Model" seems to imply that it is complete, but we need several of those structures to determine the billing.

Reply:
Yes indeed, I renamed it